proxy/test/TokenManagerERC20.ts

```diff
@@ -153,7 +154,7 @@ describe("TokenManagerERC20", () => {
 
     it("should reject with `Insufficient funds` if token balance is too low", async () => {
         // preparation
-        const error = "Insufficient funds";
+        const error = "insufficient funds";
```

Comment:
Looks like it would not pass

Reply:
Solidity revert message with Upper capital letter

Reply:
It passes. (-:
I'm not sure why but more detailed error message are raised here. To match it I had to change capitalization.

Reply:
Interesting